cli/cli-improvements.md

```diff
@@ -0,0 +1,330 @@
+# CLI improvements
+
+* **Status**: Draft
+* **Author**: Nithya Subramanian (@nithyatsu)
+
+## Overview
+
+When a Radius user wants to deploy an application, a few concepts come into picture
+
+*	Resource group – Radius resource group mimic ARM resource group and are tied to RBAC. 
+*	Environment – Radius environment is what the application uses. The environment can have a set of recipes and is “in” a resource group itself, since it requires RBAC to make sure not everyone can edit it and for example, point a Production environment to use Sandbox recipes.
+*	Workspace – Radius workspace has a "per user" information about the kubernetes context/ cluster where the Radius services are hosted as well as the “default” environment and resource group  of the user.
```

Comment:
if it's "per user" then does it mean the user can customize these things? what and how can they customize?